Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.py

```diff
@@ -3340,7 +3339,9 @@ def fetch_relevant_tickets(
         tickets_capacity -= tickets_amount
 
         if fetched_tickets:
-            last_ticket_create_time = total_tickets[-1].get("CreateDate")
+            last_ticket_create_time = max(
+                [date_to_epoch_for_fetch(arg_to_datetime(ticket.get("CreateDate")))
+                 for ticket in total_tickets])
```

Comment:
won't we run now date_to_epoch_for_fetch twice?

Reply:
yes, thanks will fix it